docs/web3-eth-ens.rst

```diff
@@ -786,7 +786,7 @@ Example
         'ethereum.eth',
         'QmXpSwxdmgWaYrgMUzuDWCnjsZo5RxphE3oW7VhTMSCoKK',
         {
-            from: '0x9CC9a2c777605Af16872E0997b3Aeb91d96D5D8c'
+            from: '0xe301017012208cd82588c4e08268fa0b824caa93847ac843410076eeedc41d65fb52eccbb9e6'
```

Comment:
The options property from should be a valid ethereum address

Reply:
My bad I mixed 2 lines. Fixed